src/platform/amd/common/include/platform/ipc.h

```diff
@@ -11,13 +11,13 @@
 #include <stdint.h>
 #include <platform/platform.h>
 
-inline uint32_t sof_ipc_host_status(void);
-inline uint32_t sof_ipc_host_ack_flag(void);
-inline void sof_ipc_host_ack_clear(void);
-inline uint32_t sof_ipc_host_msg_flag(void);
-inline void sof_ipc_host_msg_clear(void);
-inline void sof_ipc_dsp_ack_set(void);
-inline uint32_t sof_ipc_dsp_status(void);
-inline void sof_ipc_dsp_msg_set(void);
+uint32_t sof_ipc_host_status(void);
+uint32_t sof_ipc_host_ack_flag(void);
+void sof_ipc_host_ack_clear(void);
+uint32_t sof_ipc_host_msg_flag(void);
+void sof_ipc_host_msg_clear(void);
+void sof_ipc_dsp_ack_set(void);
+uint32_t sof_ipc_dsp_status(void);
+void sof_ipc_dsp_msg_set(void);
```

Comment:
right, just please avoid `inline` in the previous commit

Reply:
Done